content/300-guides/200-deployment/110-deployment-guides/550-deploying-to-deno-deploy.mdx

```diff
@@ -6,141 +6,166 @@ metaDescription: 'Learn how to deploy a TypeScript application to Deno Deploy.'
+### Result
+
+With this command, you achieve 2 results. 
+
+* install Prisma Client in your project
+* create `package.json` file
 
 ## 3. Generate a Prisma Client for Data Proxy
 
-Next, generate a Prisma Client that connects through the [Data Proxy](/data-platform/data-proxy) over HTTP.
+Next, generate Prisma Client for the Data Proxy. You use the [Data Proxy](/data-platform/data-proxy) to connect to your database over HTTP.
 
 ```terminal
 deno run -A --unstable npm:prisma generate --data-proxy
 ```
 
 ## 4. Create your application and test it locally
 
-You now have a database schema and a generated local Prisma Client. You can now create a minimal local Deno application. Create an `index.ts` with the following content:
+You now have a database schema and a generated local Prisma Client. You can now create a local Deno application.
+
+### Steps
+
+Create an `index.ts` with the content listed below.
 
-```ts
-import { serve } from 'https://deno.land/std@0.140.0/http/server.ts'
-import { PrismaClient } from './generated/client/deno/edge.ts'
+  ```ts
+  import { serve } from 'https://deno.land/std@0.140.0/http/server.ts'
+  import { PrismaClient } from './generated/client/deno/edge.ts'
 
-const prisma = new PrismaClient()
+  const prisma = new PrismaClient()
 
-async function handler(request: Request) {
-  const log = await prisma.log.create({
-    data: {
-      level: 'Info',
-      message: `${request.method} ${request.url}`,
-      meta: {
-        headers: JSON.stringify(request.headers),
+  async function handler(request: Request) {
+    const log = await prisma.log.create({
+      data: {
+        level: 'Info',
+        message: `${request.method} ${request.url}`,
+        meta: {
+          headers: JSON.stringify(request.headers),
+        },
       },
-    },
-  })
-  const body = JSON.stringify(log, null, 2)
-  return new Response(body, {
-    headers: { 'content-type': 'application/json; charset=utf-8' },
-  })
-}
+    })
+    const body = JSON.stringify(log, null, 2)
+    return new Response(body, {
+      headers: { 'content-type': 'application/json; charset=utf-8' },
+    })
+  }
 
-serve(handler)
-```
+  serve(handler)
+  ```
 
-You cannot execute this script yet, because you lack the credentials for Data Proxy that let you use this Client with your database. In the next section, you will resolve this.
+### What's next
+
+You cannot run this script yet, because you do not yet have the required Data Proxy credentials to use Prisma Client with your database. You obtain the required credentials when you [add your application to the Prisma Data Platform](#6-create-your-project-in-the-prisma-data-platform).
 
 ## 5. Create a repository and push to GitHub
 
-To prepare for the steps ahead, [create a private repository](https://github.com/new) on GitHub.
+Before you add your application to the Prisma Data Platform, create a private repository for your application on GitHub.
```

Comment:
Now that the PDP does not require a Github repo any more, this step could in theory be done later only when needed for Deno Deploy. Not sure if it makes a difference.

Reply:
That's true. I've removed this section as it no longer makes sense for this guide.